server/bin/deploy

```diff
@@ -336,34 +334,6 @@ else
     HOSTEDTEST=""
 fi
 
-# Delete the allmsg Qpid queue. The error messages produced during
-# deletion are silenced
-# This deletion is attempted regardless of whether Qpid is enabled. We do that
-# to prevent the queue to grow indefinitely
-QPID_KEYS_DIR=$PROJECT_DIR/../server/bin/qpid/keys
```

Comment:
We should also remove the creation of the `/etc/candlepin/certs/amqp` directory from the spect.tmpl file.
Specifically these lines seem to be irrelevant now:
```
%{__install} -d -m 755 %{buildroot}/%{_sysconfdir}/%{name}/certs/amqp/

and

## amqp dir, keystore & truststore used by qpid JMS client
%ghost %attr(644, tomcat, tomcat) %{_sysconfdir}/%{name}/certs/amqp/keystore
%ghost %attr(644, tomcat, tomcat) %{_sysconfdir}/%{name}/certs/amqp/truststore
```

Reply:
Removed.